### PR TITLE
remove in workflow update

### PR DESCRIPTION
**Pull request by zhongjis**
- fix README.md fix line breakers
- remove update options in `.bwconf` 1. remove option code 2. remove update icons 3. update readme.md
- add `pref.plist` to `.gitignore`

### CODE REVIEW

**Comment by zhongjis** [comment]
@blacs30 
for some reason the change does not reflect on my new build generated using `make universal-binary`. wondering if you got any insight on this. does this relate to my go setting?


Screenshot 2023-06-17 at 12 05 10

**Comment by zhongjis** [comment]
tuned out my build was having some typos. now it should be fixed
Screenshot 2023-06-24 at 12 58 03

**Comment by zhongjis** [comment]
@blacs30 let me know if im missing anything

**Review by blacs30 — CHANGES_REQUESTED** [review]
sorry, haven't submitted my review when I made it

File: src/cli.go [hunk 1]
```diff
@@ -279,7 +263,7 @@ func runConfig() {
 		UID("icons").
 		Icon(iconReload).
 		Var("action", "-icons").
-		Var("notification", "Downloading Favicons for URLs").
+		Vas("notification", "Downloading Favicons for URLs").
```
[the remark this answers is not in the document]
Comment (zhongjis) [reply src/cli.go]: seems like I forgot to push my latest changes. it is fixed

**Review by blacs30 — APPROVED** [review]

**Pull request merged**